zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Table.java

```diff
@@ -68,24 +68,26 @@ public void createTable(HttpServletRequest request, HttpServletResponse response
             response.setContentType(contentType);
         }
 
-        Resource resource = collectionReader.getResource(uri);
-        TableModifications modifications = getTableModifications(request);
-        Node table = XlsToHtmlConverter.convertToHtmlPageWithModifications(resource.getData(), modifications);
-        String output = XlsToHtmlConverter.docToString(table);
-
-        // Write the file to the response
-        org.apache.commons.io.IOUtils.copy(new StringReader(output),
-                response.getOutputStream());
-
-        getCollectionHistoryDao().saveCollectionHistoryEvent(collection, session, COLLECTION_TABLE_CREATED,
-                tableCreated(uri, modifications));
-
-        Audit.Event.COLLECTION_TABLE_CREATED
-                .parameters()
-                .host(request)
-                .collection(collection)
-                .user(session.email)
-                .log();
+        try (Resource resource = collectionReader.getResource(uri)) {
+
+            TableModifications modifications = getTableModifications(request);
+            Node table = XlsToHtmlConverter.convertToHtmlPageWithModifications(resource.getData(), modifications);
```

Comment:
I did see that - I did a google search to try and work out what should be closed but couldn't find an answer. Their API doc doesn't say anything either.